### PR TITLE
[OXT-309] qemu: maintain guest time offset

### PR DESCRIPTION
**Pull request by cjp256**

### CODE REVIEW

File: recipes-openxt/qemu-dm/files/0001-generic-xenstore-extensions.patch [hunk 1]
```diff
@@ -54,9 +54,11 @@ Chris Patterson, <pattersonc@ainfosec.com>, 03/31/2015
  hw/xen_backend.h |    4 ++++
```
Comment (rianquinn) [inline recipes-openxt/qemu-dm/files/0001-generic-xenstore-extensions.patch]: Missing header :smile:

Comment (cjp256) [inline recipes-openxt/qemu-dm/files/0001-generic-xenstore-extensions.patch]: No it's not. Go look at the patch.

**Comment by jean-edouard** [comment]
Not a blocker here, since you're not really modifying its content, but `0024-Stubdom-support-for-ISO-media-changes-after-boot.patch` really ought to be ported to the new CONFIG_STUBDOM way of building qemu-dm-stubdom and moved to qemu-dm.inc...

**Comment by cjp256** [comment]
Agreed, however, that is well outside the scope of this ticket/PR :)

**Comment by jean-edouard** [comment]
I think you shouldn't update it and let the build break on it :) jk

File: recipes-openxt/qemu-dm/files/0001-generic-xenstore-extensions.patch (hunk 1 above)
Comment (rianquinn) [inline recipes-openxt/qemu-dm/files/0001-generic-xenstore-extensions.patch]: Rats.... thought I got you there. Better luck next time !!!

**Comment by cjp256** [comment]
To test: start Windows guest.  Change timezone (to something non-UTC), note time displayed in Windows.  Reboot.  Time should be consistent after reboot.

Without this fix, time will revert to the system UTC time (e.g. off by timezone offset).  (Unless you applied registry change such as http://crashmag.net/configuring-windows-7-support-for-utc-bios-time)

**Comment by rossphilipson** [comment]
Tested - works fine. +1

Merging.

**Pull request merged**